src/Tvheadend.h

```diff
@@ -60,6 +60,9 @@ extern "C" {
 #define FAST_RECONNECT_INTERVAL   (500) // ms
 #define UNNUMBERED_CHANNEL      (10000)
 #define INVALID_SEEKTIME           (-1)
+#define DEFAULT_WEIGHT            (150)
```

Comment:
I'd use something like `enum SubscriptionWeight` in `CHTSPDemuxer` instead of defines.